connector/protobuf/README.md

```diff
@@ -0,0 +1,37 @@
+# Spark Protobuf - Developer Documentation
+
+## Getting Started 
+
+### Build
+
+```bash
+./build/mvn -Phive clean package
```

Comment:
For `protobuf` module, do we have to compile with `-Phive`?

Reply:
It doesn't seem to be necessary. Let me adjust it.